### ISSUE TITLE
Panel titles should be read out as tabs not buttons

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52017]
Environment Details:
VSCode Version : 1.24.0
OS Version : Win10

Additional Details:
MAS Violated: MAS1.3.1
Mac Version: HighSierra10.13.5 
Jaws Version: 18.0.5038

Repro Steps:

1.Launch VS Code while NVDA is ON.
2.Go to View menu and Select Problems.

Actual:
NVDA reads all the tab items (Problems, Output, Debug Console, Terminal) as buttons. Same Issue has also been observed with Jaws and Voice Over.

Expected:
NVDA should read it as Tab item.

Recommendations:
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

User Impact:
User will not get the clear understanding of data if those controls will be announced as buttons.

MAS Reference:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

Attachment for Reference: 
[A11y VSCode Problems NVDA announces button.pptx](https://github.com/Microsoft/vscode/files/2105729/A11y.VSCode.Problems.NVDA.announces.button.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
@JacksonKearl let me know if you might be interested in this
Assigning also to @sandy081 as this is coming from the composite bar which uses the action bar
https://github.com/Microsoft/vscode/blob/fe82cb86cbcf606e0d7d0c218a988edf3acd828a/src/vs/workbench/browser/parts/compositebar/compositeBar.ts#L72
